src/home/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="toolbar">
+        <a href="#">My Logo</a>
+        <nav>
+          <img src="../assert/images/toggle.png" alt="Toggle" />
+        </nav>
+      </header>
+      <section class="headline">
+        <p class="heading">I am Mwafrika Glad to see you!</p>
+        <p class="description">

```

Comment:
- [ ] Please, consider, that you have to create two sections: toolbar (header) and headline section. In `body` tag should be two sections. Considering your code, you did correctly `header` tag and `section` tag, but you inserted them into a whole `div` tag, aka you should use HTML5 semantic tags.

Reply:
Do you mean, I must replace the header tag with a section? also for the div containing the whole stuff, should I remove it completely? or just rename it as a section?

Reply:
I am done with the changes. I am waiting for approval. Thanks.

Reply:
> I am done with the changes. I am waiting for approval. Thanks.

Please, submit again your PR for the next review.

Reply:
> Thank you for the feedback, Do you mean, I must replace the header tag with a section? also for the div containing the whole stuff, should I remove it completely? or just rename it as a section?

You should just delete `div` section and keep  `header` and `section` tags.

Reply:
Yes. I am done with that. I have resubmitted and I am waiting for the next review.